Functions/BIDS/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -542,13 +542,24 @@
 end
 
 % First count the number of controls, labels, and deltaMs
+% Check and exclude dummy scans first from ASLContext
```

Comment:
Are we sure we also removed the dummy scans from the image volume, if we remove it from the ASLContext here? Or am I mixing things up

Reply:
Yes. You are mixing things up :) Actually, a good comment. This part here is about counting controls/labels, so we are removing the dummy scans here only for the calculation so that we can calculate correctly TotalAcquiredPairs.

The real removal of dummy volumes are done in another function.

Reply:
OK but still we should link them in another issue right. And what is BIDS? TotalAcquiredPairs should normally be all volumes, including the dummy?

Reply:
Will be all solved in #1456